src/markdown/developers.md

```diff
@@ -24,3 +24,54 @@ https://www.glimmer.tech
 #### Bitforest
 
 http://www.bitforest.co
+
+---
+
+### [Submit Your Game!]
```

Comment:
remove these brackets, it is making an invalid markdown file, unless do you want to make this a link? then just add 
`### [Submit Your Game!](/games)`